ThunderCloud/QuizBadgeScrollerViewCell.swift

```diff
@@ -68,6 +68,7 @@ open class QuizBadgeScrollerViewCell: CollectionCell {
 			} else {
 				parentViewController?.present(shareViewController, animated: false, completion: nil)
 			}
+            
```

Comment:
Is this blank line needed?

Reply:
Aye, it should be here because this is a multi-line if statement with an else below it ☺️